MAUI/Scheduler/migration.md

```diff
@@ -2454,6 +2470,104 @@ this.Content = scheduler;
 
 </table> 
 
+### DragDropSettings
+
+The following code example explains how to customize the drag and drop operations of appointments in schedule.
+
+<table>
+<tr>
+<th>Xamarin SfSchedule</th>
+<th>.NET MAUI SfScheduler</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+<schedule:SfSchedule AllowAppointmentDrag="True" >
```

Comment:
check syntex error
schedule: to scheduler:

Reply:
Schedule is used for existing xamarin forms  codes